applications/dashboard/controllers/api/MediaApiController.php

```diff
@@ -158,6 +165,11 @@ public function get($id) {
         }
 
         $row = $this->normalizeOutput($row);
+        $ext = pathinfo($row['url'], PATHINFO_EXTENSION);
+        if (!in_array($ext, array_keys(ImageResizer::getExtType()))) {
```

Comment:
Removing the width and height fields should probably go in the `normalizeOutput` method. Right now, these fields are still present in post responses.